### PR TITLE
Add virtual host config.

### PR DESCRIPTION
**Pull request by chadwcarlson**
* Adds virtual host configuration for Rabbit MQ.
* Cleans up headings on that page.

### CODE REVIEW

**Comment by Crell** [comment]
Does this addition have any implication for the relationships array, like it does for MySQL, or is it all on the client?

File: src/configuration/services/rabbitmq.md [hunk 1]
```diff
@@ -83,3 +83,19 @@ dependencies:
 ```
 
 Then, when you SSH into your container, you can simply type any `amqp-` command available to manage your queues.
+
+## Configuration
+
+### Virtual hosts
+
+You can configure additional [virtual hosts](https://www.rabbitmq.com/vhosts.html) to a RabbitMQ service, which can be useful for separating resources. In your `.platform/services.yaml` file define the names of the virtual hosts under the `configuration.vhosts` attribute:
```
Comment (JGrubb) [inline src/configuration/services/rabbitmq.md]: Can we flesh this out a tiny bit more?  "Useful for separating resources" but why would you want to do that?

Comment (ralt) [reply src/configuration/services/rabbitmq.md]: It's like, yes, you can use prefix on your mysql tables to use a single database, but the correct abstraction is using several databases. Same idea for rabbitmq vhosts. Not sure we need to expand on this.

**Comment by ludoge** [comment]
This feature has been deployed to all live regions.

**Comment by ralt** [comment]
@Crell no implications there

**Pull request merged**